single_upload.py

```diff
@@ -30,20 +30,10 @@
 
 videoList = []
 for fileName in dirList:
-    if fileName.endswith(".mp4"):
-        videoList.append(fileName)
-    if fileName.endswith(".m4a"):
-        videoList.append(fileName)
-    if fileName.endswith(".3gp"):
-        videoList.append(fileName)
-    if fileName.endswith(".avi"):
-        videoList.append(fileName)
-    if fileName.endswith(".mov"):
-        videoList.append(fileName)
-    if fileName.endswith(".wmv"):
+    if fileName[-3:] in ["mp4","m4a","3gp","avi","mov","wmv"]:
```

Comment:
So we'd have to update the conditional as soon as you want to check against a file extension that is longer than 3 characters, but in general it is good to have DRY (Don't Repeat Yourself) code.